app/src/main/res/raw/auth_config_multiple_account.json

```diff
@@ -1,7 +1,7 @@
 {
-  "client_id" : "<enter_your_clientID>",
-  "authorization_user_agent" : "WEBVIEW",
-  "redirect_uri" : "<enter_your_redirect_uri>",
+  "client_id" : "0984a7b6-bc13-4141-8b0d-8f767e136bb7",
+  "authorization_user_agent" : "DEFAULT",
+  "redirect_uri" : "msauth://com.azuresamples.msalandroidapp/1wIqXSqBj7w%2Bh11ZifsnqwgyKrY%3D",
```

Comment:
These changes should be reverted, and only added to the branch we use in the pipeline